java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -513,6 +514,72 @@ void testReadParquetFull() {
     }
   }
 
+  @Test
+  void testReadORC() {
+    assumeTrue(Cuda.isEnvCompatibleForTesting());
+    ORCOptions opts = ORCOptions.builder()
+            .includeColumn("float1")
+            .includeColumn("boolean1")
+            .includeColumn("byte1")
+            .build();
+    try (Table table = Table.readORC(opts, TEST_ORC_FILE)) {
+      long rows = table.getRowCount();
+      assertEquals(2, rows);
+      assertTableTypes(new DType[]{DType.FLOAT32, DType.BOOL8, DType.INT8}, table);
```

Comment:
Rather than just checking the row count and types, the test should build the expected table and compare the loaded table with the expected table.  See `testReadCSV` for an example.